README.md

```diff
@@ -1,4 +1,56 @@
-# ActiveModelSerializers
+# ActiveModelSerializers (Deprecated)
```

Comment:
I updated the description for some clarity in my intent.

like, it sucks, cause it has high visibility, but @beauby did a GREAT job with jsonapi-rb. 

I just don't feel like _another_ re-write is going to give people confidence. 
migrating to jsonapi-rb is pretty straight forward, too. 

tonight, I'm working on some compatibility stuff to allow smooth migration from AMS to jsonapi-rb.